include/cocaine/rpc/session.hpp

```diff
@@ -89,8 +98,21 @@ class session_t:
     void
     pull();
 
+    // TODO: Do something with it.
+    template<class Event, class... Args>
     void
-    push(io::encoder_t::message_type&& message);
+    push(uint64_t channel_id, Args&&... args) {
```

Comment:
There is a TODO several lines above :)

AFAIK currently it is required by encoder (which owns the header table) for its purposes related with header table.